### PR TITLE
5 add tags option to snippets

### LINKED ISSUE #5: Add tags option to snippets
We need tags props on the snippets, it will help us to implement search and group functionality later.

### PR DESCRIPTION
**Pull request by olierxleben**
Added tag handling to Astro files and displaying them on index and snippets page. I added defensive handling in BlogCard (if no tags are set, no tags are displayed. 

image

comments or suggestions are very welcome

closes #5

### CODE REVIEW

File: src/pages/models.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export interface Frontmatter {
+  title: string;
+  pubDate: string;
+  description: string;
+  contributedBy: string;
+  tags: string;
```
Comment (santoshyadavdev) [inline src/pages/models.ts:6]: Can we make an array of string

**Comment by santoshyadavdev** [comment]
can we also show the tags on snippet page as well after date

**Comment by olierxleben** [comment]
> can we also show the tags on snippet page as well after date

image

**Comment by santoshyadavdev** [comment]
> > can we also show the tags on snippet page as well after date
> 
> image

This is nice, thanks you, lets add it on next line 
<date><Twitter Handle>
<Tags>

WDYT?

**Comment by olierxleben** [comment]
> > > can we also show the tags on snippet page as well after date
> > 
> > 
> > image
> 
> This is nice, thanks you, lets add it on next line date Twitter Handle Tags
> 
> WDYT?

Good idea. :)

**Comment by santoshyadavdev** [comment]
Thank you, This looks great.

**Review by santoshyadavdev — APPROVED** [review]

**Pull request merged**